packages/api/src/services/emp-registry.ts

```diff
@@ -39,7 +39,10 @@ export default async (config: Config, appState: Dependencies, emit: (event: Even
       const blockNumber = contracts[address].blockNumber;
       registeredEmpsMetadata.set(address, { blockNumber });
       registeredEmps.add(address);
-      emit("created", { address, blockNumber, startBlock, endBlock });
+
+      if (emit) {
```

Comment:
might consider just setting a default function that doesnt do anything in the constructor if one is not supplied

Reply:
provided a default value for the parameter